.atom-build.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
This file shouldn't be here, I guess... I use Atom as well, but this file shouldn't be in the source repo.